MonumentAddons.cs

```diff
@@ -737,6 +737,42 @@ private void CommandSetDirection(IPlayer player, string cmd, string[] args)
             _adapterDisplayManager.ShowAllRepeatedly(basePlayer);
         }
 
+        [Command("maskull")]
+        private void CommandSkull(IPlayer player, string cmd, string[] args)
+        {
+            if (player.IsServer || !VerifyHasPermission(player))
+                return;
+
+            SingleEntityController controller;
+            SingleEntityAdapter adapter;
+            if (!VerifyLookingAtAdapter(player, out adapter, out controller, LangEntry.ErrorNoSuitableAddonFound))
+                return;
+
+            if (!controller.EntityData.PrefabName.Contains("assets/prefabs/misc/halloween/trophy skulls/"))
```

Comment:
Why not check for the `SkullTrophy` class here?

Reply:
Because Im dumb :)